doc/Windows.md

```diff
@@ -0,0 +1,14 @@
+# Introduction #
```

Comment:
Hmm, definitely design manual, but the contents doesn't seem very useful or being used, so delete is fine too